content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx

```diff
@@ -134,3 +134,90 @@ updateUser: (parent, args, ctx: Context) => {
   })
 },
 ```
+
+## The effect of <inlinecode>null</inlinecode> and <inlinecode>undefined</inlinecode> on conditionals
+
+There are some caveats to filtering with conditionals which might produce unexpected results. When filtering with conditionals you might expect one result but receive another given how Prisma treats nullable values.
+
+### A high level overview
+
+
+|Operator   | 0 filters              | 1 filter               | n filters            |
+|-----------|------------------------|------------------------|----------------------|
+|`OR`       | return empty list      | validate single filter | validate all filters |
+|`AND`      | return all items       | validate single filter | validate all filters |
+|`NOT`      | return all items       | validate single filter | validate all filters |
+
+The following example shows how an `undefined` parameter can impact the expected return data when passed to a conditional operator such as [`OR`](../../../reference/api-reference/prisma-client-reference#or) <span class="api"></span>.
```

Comment:
I would make this a little bit more assertive - either just change "can impact" to "impacts" and "conditional operator such as" to "the `OR` conditional operator", or something like this:

```suggestion
The following example shows how an `undefined` parameter impacts the results returned by a query that uses the [`OR`](../../../reference/api-reference/prisma-client-reference#or) <span class="api"></span> operator.
```

Reply:
Solid suggestion, thanks!